docs/patterns/lazyloading.rst

```diff
@@ -90,14 +90,22 @@ Then you can define your central place to combine the views like this::
 You can further optimize this in terms of amount of keystrokes needed to
 write this by having a function that calls into
 :meth:`~flask.Flask.add_url_rule` by prefixing a string with the project
-name and a dot, and by wrapping `view_func` in a `LazyView` as needed::
+name and a dot, and by wrapping `view_func` in a `LazyView` as needed.
+Additionally, instead of just passing in one url_rule at a time, we're going 
+to use a list to hold one to many url_rules. This will allow us to have several
+routes for a single endpoint::
 
-    def url(url_rule, import_name, **options):
+    def url(import_name, url_rules=[], **options):
         view = LazyView('yourapplication.' + import_name)
-        app.add_url_rule(url_rule, view_func=view, **options)
+        for url_rule in url_rules:
+            app.add_url_rule(url_rule, view_func=view, **options)
 
-    url('/', 'views.index')
-    url('/user/<username>', 'views.user')
+    # add a single route to the index view
+    url('views.index', ['/']  )
+
+    # add two routes to a single function endpoint
+    url_rules = ['/user/','/user/<username>']
```

Comment:
I wasn't sure what the target audience was for this one, so I figured that seeing another way might help someone that might not otherwise understand.